lib/modules/keyboardNav.js

```diff
@@ -1181,9 +1181,7 @@ module.nextGalleryImage = function() {
 	}
 };
 
-module.toggleViewImages = function() {
-	ShowImages.setShowImages();
-};
+module.toggleViewImages = ShowImages.toggleViewImages;
```

Comment:
I believe this is only used in one place (keyboard shortcut), can you move it there and get rid of `module.toggleViewImages`?